be/src/exec/pipeline/fragment_context.h

```diff
@@ -45,6 +45,13 @@ namespace pipeline {
 using RuntimeFilterPort = starrocks::RuntimeFilterPort;
 using PerDriverScanRangesMap = std::map<int32_t, std::vector<TScanRangeParams>>;
 
+// clang-format off
```

Comment:
this comment can turn clang-format off? it seems really normal.

Reply:
Yes. clang-format has bug for multiple lines of `concept`, so disable it.